dart/send-http-request/lib/main.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+import 'package:dio/dio.dart' hide Response;
+
+Future<void> start(final req, final res) async {
+
+  if (req.payload == '') {
+    res.send("No valid payload");
+    return;
+  }
+  final payload = jsonDecode(req.payload);
+
+  String url = payload['url'] ?? '';
+  // Map<String, String> headers = payload['headers'] ?? {};
+  String method = payload['method'] ?? '';
+  // String body = payload['body'] ?? '';
+
+  if (url.isEmpty || method.isEmpty) {
+    res.send("No valid payload parameter values");
+    return;
+  }
+
+  var response =
+      await Dio().fetch(RequestOptions(path: url.trim(), method: method));
+
+  if (response.statusCode != 200) {
+    res.json({'success': false, 'message': "URL could not be reached."});
```

Comment:
make sure to return after so the following `res.json()` isn't called:

```suggestion
    res.json({'success': false, 'message': "URL could not be reached."});
    return;
```

Reply:
Also, the problem might not have been because the URL could not be reached. It's probably best to use the response body instead of `"URL could not be reached."` as the `message`

Reply:
Thanks for all of your suggestions 🎇 I am working on it.